### PR TITLE
Fix: popup breaking screen

### PR DESCRIPTION
**Pull request by kushalsourav**
fixed auto close popover when the user scrolls past the app card and didn't fix right side screen breaking 

Fix: #11256

### CODE REVIEW

File: frontend/src/HomePage/AppCard.jsx [hunk 1]
```diff
@@ -90,7 +110,9 @@ export default function AppCard({
                   canUpdateApp={canUpdateApp(app)}
                   deleteApp={() => deleteApp(app)}
                   exportApp={() => exportApp(app)}
-                  isMenuOpen={isMenuOpen}
+                  isMenuOpen={setMenuOpen}
```
Comment (rudeUltra) [inline frontend/src/HomePage/AppCard.jsx:113]: Hey, @kushalsourav why was this changed to setMenuOpen

Comment (kushalsourav) [reply frontend/src/HomePage/AppCard.jsx:113]: Hey @rudeUltra  thank you for noticing this. I forgot to remove that line. The `isMenuOpen` prop is not defined as a prop in  `AppMenu.jsx`. I think we don’t need that line

* The issue was to fix the screen breaking at the bottom edge when the last row's app menu popup is visible
* I got an update on that issue: when the user clicks the button to open the pop-up, scrolling past the `App Card` linked to the pop-up closes it
* After fixing that, I realized the Home page has pagination, as users won’t be scrolling down when the screen width is at its maximum
* Popups auto close works only when screen-Y (Y-axis) is scrollable

**Comment by kushalsourav** [comment]
@rudeUltra 

https://github.com/user-attachments/assets/7d79e486-b65d-438b-a35a-7298c7e434b1

Please let me know if there are any changes or suggestions

**Review by rudeUltra — APPROVED** [review]
